### PR TITLE
feat: 优化 gitee 初始化

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Optimize gitee initialization    |
| 🇨🇳 Chinese |     优化 gitee 初始化      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: site/theme/template/Layout/Header/Navigation.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ export default ({
           </a>
         </Menu.Item>
       )}
-      {isZhCN && !isGitee && (
+      {isZhCN && isGitee !== undefined && !isGitee && (
```
Comment (zombieJ) [inline site/theme/template/Layout/Header/Navigation.tsx:112]: 好像不对，这是因为 ssr 的时候的确不是 gitee 所以它预渲染展示了这个按钮，然后在页面加载的时候因为没设置所以就又藏了起来，然后再通过 useEffect 设置了回来。所以这边看应该不需要状态控制，直接静态检查就行了。

Comment (crazyair) [reply site/theme/template/Layout/Header/Navigation.tsx:112]: 初始化是 undefined，所以这里判断 不是undefined 之后再走后面的逻辑

Comment (crazyair) [reply site/theme/template/Layout/Header/Navigation.tsx:112]: preview 后，我 LightProxy 代理成 gitee 访问测试下

Comment (crazyair) [reply site/theme/template/Layout/Header/Navigation.tsx]: 确实，没有 gitee 情况下默认不显示，我改成直接获取了

**Pull request merged**